Dockerfile

```diff
@@ -1,32 +1,34 @@
-FROM manageiq/ruby:latest
-
-RUN yum -y install centos-release-scl-rh && \
-    yum -y install --setopt=tsflags=nodocs \
-                   # To compile native gem extensions
-                   gcc-c++ \
-                   # For git based gems
-                   git \
-                   # For checking service status
-                   nmap-ncat \
-                   # To compile pg gem
-                   rh-postgresql10-postgresql-devel \
-                   rh-postgresql10-postgresql-libs \
-                   # For the RMagick gem
-                   ImageMagick-devel \
-                   && \
-    yum clean all
+FROM registry.access.redhat.com/ubi8/ubi
+
+RUN dnf -y --disableplugin=subscription-manager module enable ruby:2.5 && \
+    dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs install \
+    # ruby 2.5 via module
+    ruby-devel \
+    # build utilities
+    gcc-c++ git make redhat-rpm-config \
+    # libraries
+    postgresql-devel openssl-devel libxml2-devel \
+    #ImageMagick deps
+    autoconf libpng-devel libjpeg-devel librsvg2 && \ 
+
+    dnf clean all
+
+# Compile ImageMagick 6 from source.
+RUN git clone https://github.com/ImageMagick/ImageMagick6 --depth 1 --branch 6.9.10-90 && cd ImageMagick6 && \
```

Comment:
Talked to @lindgrenj6 about this separately, but I would rather have the source available directly rather than pulling from a URL at build time.

We tried to build our own rpm, but that wasn't working out so well.

Reply:
I think the choice was to include the tarball from the srpm directly in this repo

Reply:
Added tarball and compile-from-src steps here.